network/common/src/main/java/org/apache/spark/network/server/RpcHandler.java

```diff
@@ -54,4 +54,11 @@ public abstract void receive(
   public void connectionTerminated(TransportClient client) { }
 
   public void exceptionCaught(Throwable cause, TransportClient client) { }
+
+  /**
+    * Whether to close idle connections
+    */
+  public boolean isCloseIdleConnections() {
```

Comment:
nit: the method name sounds a little weird. maybe just `closeIdleConnections()`.

I'm also a little bit torn about putting this in `RpcHandler`. It's not necessarily tied to RPCs. How about a new constructor argument in `TransportContext`? I'd suggest a configuration, but we don't really want users to configure this (or do we?).